extensions/wikia/PhalanxII/fallback/Phalanx.class.php

```diff
@@ -414,33 +414,33 @@ static function findBlocked($text, $blocksData, $writeStats = true, &$matchingBl
 				$isCase = $blockData['case'];
 			} else { // short format
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)